### PR TITLE
Update Ansible deployment and Create offline deployment file

### PR DESCRIPTION
**Pull request by lilin90**
PTAL

### CODE REVIEW

File: op-guide/ansible-deployment.md [hunk 1]
```diff
@@ -20,48 +20,44 @@ You can use the TiDB-Ansible configuration file to set up the cluster topology,
...
+      ```
+    
+    - Access to the external network to install curl package and download binary.
+    - Access to the managed nodes via SSH using password login or SSH authorized_key login.
 
-1.2  Several managed nodes with the following requirements:
+2. Several managed nodes with the following requirements:
 
-- 4 or more machines. At least 3 instances for TiKV. Don’t deploy TiKV together with TiDB or PD on the same machine. See [deploying recommendations](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md).
+    - 4 or more machines. At least 3 instances for TiKV. Don’t deploy TiKV together with TiDB or PD on the same machine. See [deploying recommendations](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md).
```
Comment (andelf) [inline op-guide/ansible-deployment.md]: `Don’t` => `Don't`

File: op-guide/ansible-deployment.md [hunk 2]
```diff
@@ -142,138 +138,241 @@ pd_servers
...
 
-## 7. Perform Rolling Update
-The rolling update of the TiDB service does not impact the ongoing business. The environment with the minimum number of instances is as follows:
+1. Connect to the TiDB cluster using the MySQL client.
 
-+ 2 TiDB instances
-+ 3 TiKV instances
-+ 3 PD instances
+    ```
+    mysql -u root-h 172.16.10.1 -P 4000
```
Comment (andelf) [inline op-guide/ansible-deployment.md]: space between `root` & `-h`

File: op-guide/ansible-deployment.md [hunk 3]
```diff
@@ -142,138 +138,241 @@ pd_servers
...
+    ```
+    
+    > **Note**: The default port of TiDB service is 4000.
+
+2. Access the monitoring platform using a browser.
```
Comment (andelf) [inline op-guide/ansible-deployment.md]: web browser

File: op-guide/ansible-deployment.md [hunk 4]
```diff
@@ -142,138 +138,241 @@ pd_servers
...
+
+    ```
+    http://172.16.10.1:3000
+    ```
+    
+    The default account and password: `admin/admin`.
```
Comment (andelf) [inline op-guide/ansible-deployment.md]: normally we use `:` to split username and password.
`/` is also acceptable.

The reason is:

in full URL scheme:

```
protocol://username:password@hostname:port/path/to/your/resource?queryparam=value&another_query_param=value2#fragment

http://admin:admin@127.0.0.1:3000/
```

File: op-guide/offline-ansible-deployment.md [hunk 5]
```diff
@@ -0,0 +1,365 @@
+---
+title: Offline Ansible Deployment
+category: operations
+---
+
+# Offline Ansible Deployment
```
Comment (queenypingcap) [inline op-guide/offline-ansible-deployment.md]: "Offline Ansible Deployment" seems not to be a frequently used phrase. Let's change it to "Offline Deployment using Ansible"

File: op-guide/offline-ansible-deployment.md [hunk 6]
```diff
@@ -0,0 +1,365 @@
...
+  
+  rpm -ivh PyYAML*.rpm libtomcrypt*.rpm libtommath*.rpm libyaml*.rpm python-
+  babel*.rpm python-backports*.rpm python-backports-ssl_match_hostname*.rpm
+  python-httplib2*.rpm python-jinja2*.rpm python-keyczar*.rpm python-
+  markupsafe*.rpm python-setuptools*.rpm python-six*.rpm python2-crypto*.rpm
+  python2-ecdsa*.rpm python2-paramiko*.rpm python2-pyasn1*.rpm sshpass*.rpm
+  rpm -ivh ansible-2.3.1.0-1.el7.noarch.rpm
+  ```
+
+2. After Ansible is installed, you can view the version using `ansible --version`.
```
Comment (queenypingcap) [inline op-guide/offline-ansible-deployment.md]: Great move! It's a good practice that you look into the code snip and document the steps not according to the Chinese version. @lilin90

File: op-guide/offline-ansible-deployment.md [hunk 7]
```diff
@@ -0,0 +1,365 @@
...
+| | [ node_exporter-0.14.0-rc.1 ](http://download.pingcap.org/node_exporter-0.14.0-rc.1.linux-amd64.tar.gz) | 
+| | [ pushgateway-0.3.1 ](http://download.pingcap.org/pushgateway-0.3.1.linux-amd64.tar.gz) |
+| | [ alertmanager-0.5.1 ](https://github.com/prometheus/alertmanager/releases/download/v0.5.1/alertmanager-0.5.1.linux-amd64.tar.gz) |
+| | [ daemontools-0.53 ](http://oifici4co.bkt.gdipper.com/daemontools-0.53.tar.gz) |
+|**Test**| [ fio-2.16 ](https://download.pingcap.org/fio-2.16.tar.gz) | 
+|**Spark**| [ spark-2.1.1-bin-hadoop ](http://download.pingcap.org/spark-2.1.1-bin-hadoop2.7.tgz) |
+| | [ tispark-SNAPSHOT-jar-with-dependencies ](http://download.pingcap.org/tispark-SNAPSHOT-jar-with-dependencies.jar) |
+| | [ tispark-sample-data ](http://download.pingcap.org/tispark-sample-data.tar.gz) |
+
+> **Note**: Download all installation packages to the Control Machine.
```
Comment (queenypingcap) [inline op-guide/offline-ansible-deployment.md]: Move this node to the beginning of this section. This content should not be on a note. It is rather the introduction of the section.

Comment (queenypingcap) [reply op-guide/offline-ansible-deployment.md]: installation packages -> packages

File: op-guide/offline-ansible-deployment.md [hunk 8]
```diff
@@ -0,0 +1,365 @@
...
+
+## Deploy for Installation
```
Comment (queenypingcap) [inline op-guide/offline-ansible-deployment.md]: Deploy for Installation -> Install the packages

File: op-guide/offline-ansible-deployment.md [hunk 9]
```diff
@@ -0,0 +1,365 @@
...
+    
+    mv pushgateway-0.3.1.linux-amd64.tar.gz pushgateway-0.3.1.tar.gz
+    
+    mv alertmanager-0.5.1.linux-amd64.tar.gz alertmanager-0.5.1.tar.gz
+  
+    ```
+
+## Orchestrate the TiDB Cluster
+
+The file path of `inventory.ini`: `tidb-ansible/inventory.ini`
```
Comment (queenypingcap) [inline op-guide/offline-ansible-deployment.md]: : -> is:

File: op-guide/offline-ansible-deployment.md [hunk 10]
```diff
@@ -0,0 +1,365 @@
...
+
+[pd_servers:vars]
+location_labels = ["host"]
+```
+
+**Edit the parameters:**
+
+1. For multiple TiKV instances, edit the `end-point-concurrency` and `block-cache-size` parameters in `conf/tikv.yml`:
+
+    - `end-point-concurrency`: keep the number lower than CPU Vcores
```
Comment (queenypingcap) [inline op-guide/offline-ansible-deployment.md]: lower -> less

**Pull request merged**